mono/tests/Makefile.am

```diff
@@ -641,6 +641,10 @@ TESTS_IL_SRC=			\
 	dim-simple.il	\
 	dim-valuetypes.il
 
+if AMD64
```

Comment:
Add it to the PLATFORM_DISABLED_TESTS instead of conditionally adding it based on the platform. Please follow how we do it in the rest of the file 😉

Reply:
I was trying. "Platform" didn't make sense to me but AMD64 does. I'll look again try to figure out what "platform" means.